src/settings/patronBlocks/Limits/LimitsForm.js

```diff
@@ -39,19 +39,19 @@ function validation(value, min, max) {
     return errorMessage;
   }
 
-  if (isInteger(min) && !isInteger(numberValue) && !Number.isNaN(numberValue)) {
+  if (isIntegerValue && !isInteger(numberValue) && !Number.isNaN(numberValue)) {
     return errorMessage;
   }
 
   return null;
 }
 
 function limitsValidation(value) {
-  return validation(value, 1, 999999);
+  return validation(value, 0, 999999, true);
```

Comment:
@Dmitriy-Litvinenko I'm not sure if I follow what is happening here. Could you explain why we need this extra flag?

Would passing it as `true` make the validation to always return `null` based on the conditions above?

Reply:
@mkuklis  We use "onlyIntegerValueAllowed" to run check for "isInteger". Add several minor changes to make the code more readable. Could you please re-review?